packages/api-client/src/conversation/data/ConversationAccessUpdateData.ts

```diff
@@ -22,7 +22,7 @@ import {CONVERSATION_ACCESS, CONVERSATION_LEGACY_ACCESS_ROLE, CONVERSATION_ACCES
 /**@deprecated */
 export interface ConversationAccessV2UpdateData {
   access: CONVERSATION_ACCESS[];
-  access_role: CONVERSATION_LEGACY_ACCESS_ROLE;
+  access_role?: CONVERSATION_LEGACY_ACCESS_ROLE;
```

Comment:
this field was not required in v2 (we were always sending data in `access_role_v2` field)